src/main/java/pl/javastart/task/Main.java

```diff
@@ -3,6 +3,28 @@
 public class Main {
 
     public static void main(String[] args) {
+        Microphone mic1 = new Microphone("Behringer", "C-1U", 225,"condenser" , 44.1,
+                48, 16, false);
+        Microphone mic2 = new Microphone("Razer", "Seiren-X", 449, "condenser", 20,
+                40, 16, false);
 
+
+        System.out.println("Microphone 1:\n" +
```

Comment:
Checkstyle zauważył w tej linii błąd. To, co znalazł możesz podejrzeć w pull requeście z zakładce "Files changed". Polecam zainstalować sobie dodatek do intellij, żeby na bieżąco informował. Instrukcję znajdziesz tutaj https://bootcamp.javastart.pl/strona/jakosc-kodu